src/ui/CustomLink/CustomLink.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import styles from './CustomLink.module.css';
+
+interface CustomLinkProps {
+    children: React.ReactNode;
+    href: string;
+    ['string']?: string;
+}
+
+function CustomLink(props: CustomLinkProps) {
+    return (
+        <>
```

Comment:
you dont need fragment here, you have one children 
https://react.dev/reference/react/Fragment